ciao-cli/instance_test.go

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) 2016 Intel Corporation
```

Comment:
I think we could arrange for these tests to be run in travis without affecting our coverage statistics.  We could just try adding a new invocation of test-cases that leaves off the coverage:

test-cases -v -timeout 9  github.com/01org/ciao/ciao-cli